whylog/tests/tests_client/__init__.py

```diff
@@ -33,13 +34,75 @@ def test_one(self, test_name):
         output_path = path_join(path, 'expected_output.txt')
 
         whylog_base = WhylogBase(parser=parsers_path, rules=rules_path)
-        whylog_client = WhylogClient(base=whylog_base, open_path=path)
+        # whylog_client = WhylogClient(rulesbase=whylog_base, open_path=path)
 
         with open(input_path, 'r') as f:
             vim_line = f.read()
-        result = whylog_client.get_cause(vim_line)
+        # TODO call get_cause with sens...
+        # result = whylog_client.get_cause(1, vim_line)
 
         with open(output_path, 'r') as f:
             pass
             # remove this comment after implement whylog client and base
             # assert result == f.read()
+
+
+class TestBacktrackSearcher(TestCase):
+    def _get_log_file_path(self):
+        prefix_path = path_join(*path_test_files)
+        path = path_join(prefix_path, "a_few_lines.log")
+        return path
+
+    def _get_sample_offset(self, line_num):
+        """
+        Returns the correct offset of line 'line_num'
+        referring to the file a_few_lines.log construction.
+        Be careful when modifying a_few_lines.log.
```

Comment:
this is very error prone. You can't expect someone to read this comment before he modifies the sample file. Assert the amount of newlines in that file instead?